### PR TITLE
Move testing from Drone to Github Actions

### PR DESCRIPTION
**Pull request by nickmcmahon01**

### CODE REVIEW

File: .github/workflows/semver.yml [hunk 1]
```diff
@@ -5,21 +5,24 @@ on:
...
-      - id: label
+      - name: Parse the SemVer label
+        id: label
         uses: UKHomeOffice/match-label-action@v1
         with:
           labels: minor,major,patch,skip-release
           mode: singular
-      - uses: UKHomeOffice/semver-tag-action@v2
+
+      - name: SemVer tag the commit
```
Comment (LiamMacP) [inline .github/workflows/semver.yml:18]: Can the name of this be brought in line with the others?

Comment (LiamMacP) [reply .github/workflows/semver.yml:18]: Same for Line 1, cleans up the required Statuses for the repo.

File: .github/workflows/codeql-analysis.yml [hunk 2]
```diff
@@ -1,28 +1,12 @@
...
 on:
-  push:
-    branches: [ main ]
   pull_request:
-    # The branches below must be a subset of the branches above
-    branches: [ main ]
+    types: [ opened, reopened, synchronize ]
   schedule:
-    - cron: '36 12 * * 2'
+    - cron: '0 2 * * 7'
```
Comment (LiamMacP) [inline .github/workflows/codeql-analysis.yml:6]: Is this run at 2 every 7 days?

Comment (nickmcmahon01) [reply .github/workflows/codeql-analysis.yml:6]: It's 2AM Sunday instead of 12:36 (lunch) on a Tuesday

**Pull request closed without merging**